keanu-examples/bayesianMethodsForHackers/src/test/java/examples/ChallengerDisasterTest.java

```diff
@@ -22,7 +23,7 @@ public void testWhenChallengerDisasterIsRunThenBetaIsSmallAndPositive() {
         System.out.println("mapBeta " + posteriors.mapBeta);
 
         // assert
-        assertThat(posteriors.mapBeta).isBetween(0.1, 0.4);
-        assertThat(posteriors.mapAlpha).isBetween(-25.0, -5.0);
+        assertThat(posteriors.mapBeta).isCloseTo(0.25, within(0.15));
```

Comment:
Nice work with AssertJ but we usually use Hamcrest. I think AssertJ is here from the original commit by Alex. I'll change it before it goes into develop. Just an FYI :)